app/models/manageiq/providers/embedded_ansible/automation_manager/configuration_script_source.rb

```diff
@@ -8,7 +8,14 @@ class ManageIQ::Providers::EmbeddedAnsible::AutomationManager::ConfigurationScri
   default_value_for :scm_type,   "git"
   default_value_for :scm_branch, "master"
 
-  belongs_to :git_repository, :dependent => :destroy
+  belongs_to :git_repository, :autosave => true, :dependent => :destroy
```

Comment:
Just caught this re: https://github.com/ManageIQ/manageiq/pull/19027#issuecomment-514217657